tests/test_rewrite_rules/test_rewrite_rules.py

```diff
@@ -32,141 +29,134 @@ class TestEntry:
 
 def build_test_entry_set(input_url: Namespace, model: str) -> List[TestEntry]:
     return [
-        TestEntry(input_url, f"includes/{model}"),
-        TestEntry(input_url, f"includes/{model}.yaml", "text/yaml"),
-        TestEntry(input_url, f"includes/{model}.ttl", "text/turtle"),
-        TestEntry(input_url, f"includes/{model}.jsonld", "application/json"),
-        TestEntry(input_url, f"includes/{model}.shex", "text/shex"),
-        TestEntry(input_url[".context.jsonld"], f"includes/{model}.context.jsonld"),
-        TestEntry(input_url[".owl"], f"includes/{model}.owl"),
-        TestEntry(input_url["/"], f"includes/{model}/"),
+        TestEntry(input_url, f"linkml-model/docs/{model}"),
+        TestEntry(input_url, f"linkml-model/linkml_model/model/schema/{model}.yaml", "text/yaml"),
+        TestEntry(input_url, f"linkml-model/linkml_model/rdf/{model}.ttl", "text/turtle"),
+        TestEntry(input_url, f"linkml-model/linkml_model/json/{model}.json", "application/json"),
+        TestEntry(input_url, f"linkml-model/linkml_model/shex/{model}.shex", "text/shex"),
+        TestEntry(input_url[".context.jsonld"], f"linkml-model/linkml_model/jsonld/{model}.context.jsonld"),
+        TestEntry(input_url[".owl"], f"linkml-model/linkml_model/owl/{model}.owl.ttl"),
+        TestEntry(input_url["/"], f"linkml-model/{model}/"),
```

Comment:
avoid hardcoding, single-source from `linkml_runtime.linkml_model.linkml_files`

Reply:
Ah, I hadn't looked for this in linkml_runtime, yes I can do that update if this test isn't found to be redundant.
Was hoping a source for these already existed.

Reply:
Aha I see that content type part now. This should also be single sourced, s.t. the .htaccess file that defines them ( https://github.com/perma-id/w3id.org/blob/master/linkml/.htaccess ) is generated from that source and we can do a single diff of that file against the expected generated copy rather than needing to check every URL.

Beyond scope of this PR, but writing to say that having these pieces be decoupled and free floating is the source of the need for these tests and usually good to address root problems rather than testing symptoms when possible